Program.cs

```diff
@@ -47,18 +47,16 @@ class Program {
             Console.CursorTop = 2; Console.CursorLeft = 0;
             Console.WriteLine ($"Solution {i} of {sFinalSoln.Count}");
             var soln = sFinalSoln[i - 1];
-            for (int j = 0; j <= 8; j++) {
-               if (j == 0)
-                  Console.WriteLine ("┌───┬───┬───┬───┬───┬───┬───┬───┐");
-               else if (j == 8) {
-                  PlaceQueens (soln[j - 1]);
-                  Console.WriteLine ("└───┴───┴───┴───┴───┴───┴───┴───┘");
-               } else {
-                  PlaceQueens (soln[j - 1]);
-                  Console.WriteLine ("├───┼───┼───┼───┼───┼───┼───┼───┤");
-               }
+            Console.WriteLine ("┌───┬───┬───┬───┬───┬───┬───┬───┐");
+            for (int j = 0; j < 7; j++) {
+               PlaceQueens (soln[j]);
+               Console.WriteLine ("├───┼───┼───┼───┼───┼───┼───┼───┤");
             }
-            Console.ReadKey ();
+            PlaceQueens (soln[7]);
+            Console.WriteLine ("└───┴───┴───┴───┴───┴───┴───┴───┘");
+            var k=Console.ReadKey(true);
+            while (k.Key is not ConsoleKey.RightArrow and not ConsoleKey.LeftArrow) k = Console.ReadKey (true);

```

Comment:
At certain time, the Total solution is shown incorrectly.

![image](https://github.com/Nandhini-Sakthi/Training/assets/16486802/ab657160-2ab2-42ea-a162-d1a2400c2c10)